docker-compose.yml

```diff
@@ -67,6 +67,16 @@ services:
     networks:
       - backend
 
+  history:
+    build: ./backend/history-service
+    container_name: history-service
+    restart: unless-stopped
+    ports:
+      - 8005:8005
```

Comment:
@YZTangent could you verify that this is intentional over just exposing like this '"8005"' as port mapping exposes the ports outside of the network which allows bypassing the gateway

Reply:
This was written before the API gateway was implemented, I can remove this now